include/circt/Dialect/Seq/SeqOps.td

```diff
@@ -78,13 +78,13 @@ def CompRegClockEnabledOp : SeqOp<"compreg.ce",
 
   let builders = [
     OpBuilder<(ins "Value":$input, "Value":$clk, "Value":$ce,
-                   "StringRef":$name), [{
+                   "Twine":$name), [{
```

Comment:
Given the eventual api this calls is a stringref, I don't know that this buys much.  Potentially stringref -> twine -> stringref conversion might require an additional string allocation.

Reply:
I generally find it annoying that StringRef functions don't take Twine arguments. I like to be able to compose strings directly in a function call without having to do the whole `Twine($expr).strref()` and just be able to pass `$expr`.

Reply:
However, this isn't necessary for this PR, and this makes this one builder inconsistent with all others.

Reply:
Removed.